assets/models/system/facebook-bart-large-cnn/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/facebook-bart-large-cnn/3d22493/mlflow_model_folder
+  container_path: huggingface/facebook-bart-large-cnn/1703165534/mlflow_model_folder
   storage_name: automlcesdkdataresources
   type: azureblob
```

Comment:
This is still pointing to the version with mlflow version 2.3.1, please save the right model

Reply:
Updated mlflow_version: 2.8.1 with change Blob Link